tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -222,5 +294,70 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 				return releaseCreated.IsReleased()
 			}, releaseCreationTimeout, defaultInterval).Should(BeTrue())
 		})
+
+		It("validates that imageIds from task create-pyxis-image exist in Pyxis.", func() {
+
+			isImageExist := 0
+			for _, imageId := range imageIDs {
+				Eventually(func() bool {
+					url := fmt.Sprintf("%s%s", pyxisStageURL, imageId)
+
+					// Create a TLS configuration with the key and certificate
+					cert, err := tls.X509KeyPair(pyxisCertDecoded, pyxisKeyDecoded)
+					if err != nil {
+						fmt.Println("Error creating TLS certificate and key:", err)
+						Expect(err).NotTo(HaveOccurred())
+					}
+
+					// Create a client with the custom TLS configuration
+					client := &http.Client{
+						Transport: &http.Transport{
+							TLSClientConfig: &tls.Config{
+								Certificates: []tls.Certificate{cert},
+							},
+						},
+					}
+
+					// Send GET request
+					request, err := http.NewRequest("GET", url, nil)
+					if err != nil {
+						fmt.Println("Error creating GET request:", err)
+						Expect(err).NotTo(HaveOccurred())
+					}
+
+					response, err := client.Do(request)
+					if err != nil {
+						fmt.Println("Error sending GET request:", err)
+						Expect(err).NotTo(HaveOccurred())
+					}
+
+					defer response.Body.Close()
+
+					// Read the response body
+					body, err := io.ReadAll(response.Body)
+					if err != nil {
+						fmt.Println("Error reading response body:", err)
+						Expect(err).NotTo(HaveOccurred())
+					}
+
+					sbomImage := &release.Image{}
+					err = json.Unmarshal(body, sbomImage)
+					if err != nil {
+						fmt.Println("Error json unmarshal body sbomPurl:", err)
+						Expect(err).NotTo(HaveOccurred())
+					}
+
+					if sbomImage.ContentManifestComponents == nil {
+						fmt.Println("Content Mainfest Components is empty.")
+						return false
+					}
+
+					isImageExist = isImageExist + len(sbomImage.ContentManifestComponents)
+
+					return isImageExist > 1
```

Comment:
```suggestion
					return len(sbomImage.ContentManifestComponents) > 0
```

Reply:
would it be better to create a method in release controller and put this code there?